package.json

```diff
@@ -14,16 +15,17 @@
 		"generate-sequelize-typediff": "npx tsx tools/toSequelizeTypescript/type-differ.ts",
 		"api-dev": "concurrently \"npm run watch-server\" \"NODE_PATH=./dist/server/client:./dist/server:./dist node --enable-source-maps init.js --watch ./dist/server --watch ./node_modules/@pubpub\"",
 		"api-prod": "NODE_PATH=./dist/server/client:./dist/server:./dist node --enable-source-maps init.js",
+		"build-client-dev": "webpack --config ./client/webpack/webpackConfig.dev.js",
```

Comment:
This is the new fast way to build the cilent using esbuild-loader. I separated it out as its own command to be able to use it in the sdk, as the client needs to be compiled in order for the app to work.